crates/oxc_parser/src/lib.rs

```diff
@@ -452,6 +454,24 @@ mod test {
         }
     }
 
+    #[test]
```

Comment:
Add to https://github.com/oxc-project/oxc/tree/main/tasks/coverage/misc instead?

Reply:
Oh it also passes before 🤔 This test is unnecessary if this is covered by codegen.